extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -1072,6 +1072,12 @@ private static void producePersistenceUnitDescriptorFromConfig(
         descriptor.getProperties().setProperty(AvailableSettings.DEFAULT_NULL_ORDERING,
                 persistenceUnitConfig.query.defaultNullOrdering.name().toLowerCase(Locale.ROOT));
 
+        if (persistenceUnitConfig.query.inClauseParameterPadding) {
```

Comment:
This `if` is not necessary, as you can use the attribute value directly

Reply:
Not sure I get what you mean